packages/ambassador/ambassador.patch

```diff
@@ -0,0 +1,10 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/ambassador/charts-original/Chart.yaml packages/ambassador/charts/Chart.yaml
+--- packages/ambassador/charts-original/Chart.yaml
++++ packages/ambassador/charts/Chart.yaml
+@@ -1,3 +1,6 @@
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: ambassador
```

Comment:
Could you please add this additional annotation for the name that will be displayed in the UI?
`catalog.cattle.io/display-name: Cakuros Ambassador`
I used Cakuros Ambassador as an example, but you can change it to whatever you think is best.

Reply:
Done!